yb-voyager/cmd/importData.go

```diff
@@ -436,28 +436,49 @@ func checkPrimaryKey(tableName string) bool {
 	}
 	defer conn.Close(context.Background())
 
-	var table, schema string
+	var table, schema, originalTableName string
 	if len(strings.Split(tableName, ".")) == 2 {
 		schema = strings.Split(tableName, ".")[0]
 		table = strings.Split(tableName, ".")[1]
 	} else {
 		schema = target.Schema
 		table = strings.Split(tableName, ".")[0]
 	}
+	originalTableName = table
```

Comment:
why is this extra variable;e required? can't you just use the `table` variable as its place.

Reply:
This variable is required for the query to check table as if there are quotes in table and we are removing them, so quoted table name is required to check if that table exist or not. It stores the original table name with quotes as well if present.